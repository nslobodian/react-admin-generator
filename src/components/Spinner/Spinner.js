import React from 'react'
import Spinner from 'react-spinner'
import './react-spinner.css'

export default () => <Spinner />
