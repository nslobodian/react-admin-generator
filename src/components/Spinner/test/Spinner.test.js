import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Spinner from '../Spinner'

describe('(Components) Spinner', () => {
  it('Should render component properly', () => {
    const wrapper = shallow(
      <Spinner />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
