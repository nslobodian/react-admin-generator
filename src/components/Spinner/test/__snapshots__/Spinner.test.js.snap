// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Components) Spinner Should render component properly 1`] = `<Spinner />`;
